modules/system/console/PluginList.php

```diff
@@ -0,0 +1,67 @@
+<?php namespace System\Console;
+
+use Illuminate\Console\Command;
+use System\Models\PluginVersion;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableSeparator;
+
+/**
+ * Console command to list existing plugins.
+ *
+ * @package october\system
+ * @author Lucas Zamora
+ */
+class PluginList extends Command
+{
+    /**
+     * The console command name.
+     * @var string
+     */
+    protected $name = 'plugin:list';
+
+    /**
+     * The console command description.
+     * @var string
+     */
+    protected $description = 'List existing plugins.';
+
+    /**
+     * Execute the console command.
+     * @return void
+     */
+    public function handle()
+    {
+        $allPlugins  = PluginVersion::all();
+
+        if (count($allPlugins) <= 0 ) {
+
+            $this->info('No plugin found');
+
+            return;
+        }
+
+        // Create a new Table instance.
+        $table = new Table($this->output);
+
+        // Set the table headers.
+        $table->setHeaders([
+            'Plugin name', 'Version','Updates enabled', 'Plugin enabled'
```

Comment:
Missing a space between `'Version',` and `'Updates`

Reply:
@LucasZdv could you add space: `'Version','Updates enabled'` -> `'Version', 'Updates enabled'`. 
Thanks!

Reply:
Needs to be fixed

Reply:
> @LucasZdv could you add space: `'Version','Updates enabled'` -> `'Version', 'Updates enabled'`.
> Thanks!

Working on it.